kubernetes-client/src/test/java/io/fabric8/kubernetes/client/informers/cache/ControllerTest.java

```diff
@@ -43,7 +43,7 @@
 import static org.mockito.Mockito.verify;
 
 class ControllerTest {
-  private final DeltaFIFO<Pod> deltaFIFO = Mockito.mock(DeltaFIFO.class, Mockito.RETURNS_DEEP_STUBS);
+  private final Store<Pod> deltaFIFO = Mockito.mock(Store.class, Mockito.RETURNS_DEEP_STUBS);
```

Comment:
nit: The name of this variable is now outdated

Reply:
I'm looking to open https://github.com/shawkins/kubernetes-client/pull/2 after this, which will address that.